recipes/main/black-pepper-tofu.md

```diff
@@ -0,0 +1,50 @@
+# Black pepper tofu
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg)
```

Comment:
This badge is broken

**Update** After a refresh it appears to be working. It's not a repeatable bug but the url looks a little weird so there might be something wrong :woman_shrugging: